sections/queueing_model/subsections/performance_measures/blocking_time/main.tex

```diff
@@ -0,0 +1,225 @@
+        c(u,v) + b(u - 1, v), & \textbf{if } v = N = T\\
+        c(u,v) + b(u, v-1), & \textbf{if } v = N \neq T \\
+        c(u,v) + p_s(u,v) b(u-1, v) + p_o(u,v) b(u, v+1), & \textbf{if } u > 0 
+        \textbf{ and } \vspace{-0.2cm} \\ 
+        & \quad v = T \\
+        c(u,v) + p_s(u,v) b(u, v-1) + p_o(u,v) b(u, v+1), & \textbf{otherwise} \\
+    \end{cases}
+\end{equation}
+
+A direct approach will be used to solve this equation here. 
+By enumerating all equations of ((\ref{eq:general_blocking_time_at_each_state})) 
+for all states \((u,v)\) that belong in \(S_b\) 
+a system of linear equations arises where the unknown variables are all the 
+\(b(u,v)\) terms. 
+Note here that these equations correspond to all blocking states as defined in
+(\ref{eq:set_of_blocking_states}). 
+Equations that correspond to non-blocking states have a value of \(0\) as 
+defined in (\ref{eq:general_blocking_time_at_each_state})
+The general form of the equation in terms of \(C,T,N \text{ and } M\) is given by: 
+
+\begin{align}
+    b(1,T) \quad &= \quad c(1, T) + p_o b(1, T + 1) \label{eq:first_eq_of_blocking_general}\\
+    b(1,T + 1) \quad &= \quad c(1, T + 1) + p_s b(1, T) + p_o b(1, T + 1) \\
+    b(1,T + 2) \quad &= \quad c(1, T + 2) + p_s b(1, T + 1) + p_o b(1, T + 3) \\
+    & \ \, \vdots \nonumber \\
+    b(1, N) \quad &= \quad c(1, N) + b(1, N - 1) \\
+    b(2, T) \quad &= \quad c(2, T) + p_s b(1, T) + p_o b(2, T + 1) \\
+    b(2, T + 1) \quad &= \quad c(2, T + 1) + p_s b(2, T) + p_o b(2, T + 2) \\
+    & \ \, \vdots \nonumber \\
+    b(M - 1, N) \quad &= \quad c(M, N - 1) + b(M, N-1) \\ 
+    b(M, T) \quad &= \quad c(T, N) + p_s b(T-1, N) + p_o b(T, N+1) \\
+    & \ \, \vdots \nonumber \\
+    b(M, N) \quad &= \quad c(M, N) + b(M, N-1) \label{eq:last_eq_of_blocking_general}
+\end{align}
+
+The equivalent matrix notation of the linear system of equations 
+((\ref{eq:first_eq_of_blocking_general})) - ((\ref{eq:last_eq_of_blocking_general}))
+is given by \(Zx=y\), where:
+\begin{equation}\label{eq:general_algebaric_approach_blocking_time}
+    \scalebox{0.73}{\(
+        Z = 
+        \begin{pmatrix}
+            -1 & p_o & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T)
+            p_s & -1 & p_o & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+1)
+            0 & p_s & -1 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+2)
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
+            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
+            0 & 0 & 0 & \dots & 1 & -1 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,N)
+            p_s & 0 & 0 & \dots & 0 & 0 & -1 & p_o & 0 & \dots & 0 & 0 \\ %(2,T)
+            0 & 0 & 0 & \dots & 0 & 0 & p_s & -1 & p_o & \dots & 0 & 0 \\ %(2,T+1)
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
+            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
+            0 & 0 & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 1 & -1 \\ %(M,N)
```

Comment:
Can we write a general form for `Z_{ij}` here please?

Reply:
So remove the matrix completely and replace it with the general form `Z_{ij}` or keep both?

Reply:
Keep both.